drivers/hmis_csv_importer/app/models/hmis_csv_importer/importer/importer.rb

```diff
@@ -197,6 +199,10 @@ def pre_process_class!(file_name, klass)
       end
     end
 
+    private def import_overrides_for(file_name)
+      HmisCsvImporter::ImportOverride.where(file_name: file_name)
```

Comment:
I might be missing something but it seems like these wouldn't be sorted so there could be non deterministic results if multiple overrides apply to the same record/col. I think there should be a sort order with id tie-breaker to avoid that. Maybe sort in app logic to make it simple:
```suggestion
      HmisCsvImporter::ImportOverride.where(file_name: file_name).
        to_a.sort_by { |r| [r.specificity, r.id]}
```
```ruby
  class ImportOverride
    def specificity
      if  matched_hud_key && replaces_value
        3
      elsif replaces_value
        2
      elsif matched_hud_key
        1
      else
        0
      end
    end
  end
  
```

Reply:
Yes, thank you!  That will be somewhat critical if we ever have less specific uses.